core/controllers/ingest.py

```diff
@@ -464,3 +464,38 @@ def parse_auth(body: dict) -> dict | None:
         except (JSONDecodeError, TypeError) as err:
             abort(400, "Invalid auth JSON", err)
     return auth
+
+
+def create_pathfinders_mappings(extracted_tables: dict[str, list[pd.DataFrame]]) -> dict[str, dict | list[str]]:
```

Comment:
As this is PF specific it doesn't feel right to keep it here. We could keep it with the transformation code as it transforms the extracted mapping tables into more usable mappings?